packages/core/test/invoke.test.ts

```diff
@@ -3009,15 +3013,6 @@ describe('invoke', () => {
           /* ... */
         })
       }
-    ],
-    [
-      'src containing a parametrized invokee with id parameter',
-      {
-        src: {
-          type: 'someSrc',
-          id: 'h4sh'
-        }
-      }
```

Comment:
any particular reason why this one was removed? it doesn't look like something that became not applicable with the change 🤔

Reply:
Sources are always strings now. There is no "invoke source definition object".

Reply:
ah, ok - makes sense now, thanks for clarifying!